api/stats/extra_mode_stats.proto

```diff
@@ -34,4 +36,64 @@ message ExtraModeGoalInfo {
     optional bool pre_items = 1;
     optional bool scored_with_item = 2;
     optional PowerUp used_item = 3;
+    optional DropshotTile dropshot_tile = 4;
```

Comment:
I am wondering if we might want to move some of these into a dropshot and rumble file.

Still have the central extra mode stats file.

It should all still be backwards compatible as long as you keep all of the same data in the same spot